resources/assets/js/store/Experience/experienceReducer.ts

```diff
@@ -493,6 +513,16 @@ export const entitiesReducer = (
           action.meta.experienceType,
         ),
       };
+    case BATCH_CREATE_EXPERIENCE_SKILLS_SUCCEEDED:
+    case BATCH_UPDATE_EXPERIENCE_SKILLS_SUCCEEDED:
```

Comment:
Careful here. The Batch Update and Create requests currently return the list of _updated_ ExperienceSkills, not _all_ ExperienceSkills. I believe `setExperienceSkills` rewrites the entire list with just the payload of this action. We don't actually don't want to delete any experienceSkills which weren't part of these batch requests.

Reply:
Actually, I believe `setExperienceSkills` is working as intended since it takes the previous `state` and spreads the `experienceSkills.byId` list with the entire list (including the ones that weren't part of the payload), then adds/overwrites the created/updated values that were part of the request.

Reply:
Somehow this ended up in the other PR, but you're right. `setExperienceSkills` looks like it adds to the state, not replaces it like I originally thought.